lib/promscrape/config.go

```diff
@@ -1197,7 +1197,7 @@ func (swc *scrapeWorkConfig) getScrapeWork(target string, extraLabels, metaLabel
 	var at *auth.Token
 	tenantID := labels.Get("__tenant_id__")
 	if len(tenantID) > 0 {
-		newToken, err := auth.NewToken(tenantID)
+		newToken, err := auth.NewTokenPossibleMultitenant(tenantID)
```

Comment:
What's the expected behavior after this change?
Do I understand it right, that if user adds `__tenant_id__ =multitenant` then it will be parsed here, but no other actions will happen and vmagent will push time series without tenant labels to multitenant remote write url?

Reply:
Main goal of this change is to keep tenantID format consistent. There is no real use cases for that.

Reply:
> Main goal of this change is to keep tenantID format consistent. There is no real use cases for tenant_id==multitenant.

What will happen with the payload if user will do such a config in labels and send it to /multitentant/? Will it fail or will get into default 0:0 tenant?

Reply:
It will get to the default `0:0` tenant according to docs:
https://docs.victoriametrics.com/cluster-victoriametrics/#multitenancy-via-labels
```
If vm_account_id or vm_project_id labels are missing or invalid, then the corresponding account ID and project ID are set to 0.
```

Reply:
I see. I don't like this change:
1. Using /multitenant/ in labels during scrape makes no practical sense, even from compatibility perspective
2. In contrary, it could result into unexpected behavior when instead of throwing an error about bad tenant settings that data will be ingested.

I'd recommend reverting changes to promscrape, I don't see how it improves consistency with other functionality. It just becomes more confusing for me.

Reply:
I agree. Update PR.